src/master-playlist-controller.js

```diff
@@ -363,10 +327,9 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
-  useAudio(label) {
-    let audioEntry;
-    let newAudioPlaylistLoader;
+  useAudio(label = null) {
     let mediaGroupName = this.masterPlaylistLoader_.media().attributes.AUDIO;
+    let audioEntries = this.masterPlaylistLoader_.master.mediaGroups.AUDIO[mediaGroupName];
```

Comment:
We need to guard against there being no `attributes` or `mediaGroups` properties in the two assignments above.